src/mapping/nav2_map_server/include/nav2_map_server/map_reps/occupancy_grid.hpp

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) 2018 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef NAV2_MAP_SERVER__MAP_REPS__OCCUPANCY_GRID_HPP_
+#define NAV2_MAP_SERVER__MAP_REPS__OCCUPANCY_GRID_HPP_
+
+#include <memory>
+#include <string>
+
+#include "nav_msgs/msg/occupancy_grid.hpp"
+#include "nav_msgs/msg/map_meta_data.hpp"
+#include "nav_msgs/srv/get_map.hpp"
+#include "rclcpp/rclcpp.hpp"
+#include "nav2_map_server/base_map_loader.hpp"
+
+enum MapMode
+{
+  TRINARY,
+  SCALE,
+  RAW
+};
+namespace nav2_map_server
```

Comment:
I'd really suggest this as two namespaces.  Hopefully that might be a convention throughout the nav2 stack.
```
namespace nav2
{
namespace map_server
{
  // ...
}
}
```

Reply:
I'll bring this up with our team